syntax/index.html

```diff
@@ -587,15 +578,16 @@
                 <p>Parse the tabular data file, using <code>DD</code> as a guide, to create a basic tabular data model (<code>T</code>) and extract <a>embedded metadata</a> (<code>EM</code>), for example from the <a>header line</a>.</p>
                 <p class="note">This specification provides a non-normative definition for parsing CSV-based files, including the extraction of <a>embedded metadata</a>, in <a href="#parsing" class="sectionRef"></a>. This specification does not define any syntax for embedded metadata beyond this; whatever syntax is used, it's assumed that metadata can be mapped to the vocabulary defined in [[!tabular-metadata]].</p>
```

Comment:
Embedded metadata is used to determine table compatibility between `TM` and `EM`. Compatibility is defined in the metadata document. Embedded metadata does not override anything; we determine that it is compatible, and then process the CSV. Note that DD is modified, and `lang` may be updated in `TM`.